scripts/deckdebug.sh

```diff
@@ -2,6 +2,22 @@
 # Usage: deckdebug.sh DECKIP:8081
 # Dependencies: websocat jq curl chromium
 
+if [ "$#" -ne 1 ]; then
+    echo "Error: Missing or incorrect argument." >&2
+    echo "Usage: deckdebug.sh DECKIP:8081" >&2
+    exit 1
+fi
+
+required_dependencies=(websocat jq curl chromium)
+
+# Check if the dependencies are installed
+for cmd in "${required_dependencies[@]}"; do
```

Comment:
This needs to be moved to after the Nix relaunch

Reply:
Doesn't `nix` also require these deps to be installed? _disclaimer; i have no idea what nix is_

Reply:
no, that nix shell thing relaunches the shell script in a temporary environment that has those dependencies installed for you